test/test_async_repository.py

```
@@ -0,0 +1,280 @@
+                collection_name = "spams"
+
+        with pytest.raises(Exception):
+            BrokenRepository(database=database)
+
+    @pytest.mark.asyncio
+    async def test_invalid_model_collection_name(self, database):
+        class BrokenRepository(AbstractRepository[Spam]):
+            class Meta:
+                collection_name = None
+
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)